plutus-context-builder/README.md

```diff
@@ -2,6 +2,12 @@
 
 ## What is this?
 
+A utility library for cteating `ScriptContext` with `Spending` or `Minting` `ScriptPurpose`. That `ScriptContext` can be used with `tasty-plutus` or by itself.
```

Comment:
> for cteating

Here is a typo.

Reply:
It can also be re-worded a bit more naturally. I suggest something like this:

A utility library for creating `ScriptContext`s with `Spending` or `Minting` `ScriptPurpose`. This provides a builder abstraction for putting `ScriptContext`s together, for use in `tasty-plutus`, or somewhere else.

Reply:
This provides a builder abstraction for assembling ScriptContext, for use in `tasty-plutus`, or somewhere else. This one?